src/runtime/Template.cpp

```diff
@@ -151,5 +156,9 @@ void Template::postProcessing(Object* instantiatedObject)
     if (m_instanceExtraData) {
         instantiatedObject->setExtraData(m_instanceExtraData);
     }
+
+    if (m_cachedObjectStructure.m_inlineCacheable) {
+        instantiatedObject->markAsNonInlineCachable();
```

Comment:
Why did you add this part?
It seems that `m_cachedObjectStructure.m_inlineCacheable` is always set to true by `Template::constructObjectStructure`, so that every instiatiated objects would be non-inlinecachable. Is it right?

Reply:
Oh. this is huge mistake!